plugins/modules/ndo_synce_interface_policy.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Anvitha Jain (@anvjain) <anvjain@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+import copy
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_synce_interface_policy
+short_description: Manage syncE Interface Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage syncE Interface Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.1 (NDO v4.3) and later.
+author:
+- Anvitha Jain (@anvjain)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be a fabric policy template.
+    type: str
+    required: true
+  inteface_policy:
+    description:
+    - The name of the syncE Interface Policy.
+    type: str
+    aliases: [ name ]
+  interface_policy_uuid:
+    description:
+    - The uuid of the syncE Interface Policy.
+    - This parameter is required when the O(interface_policy) needs to be updated.
+    type: str
+    aliases: [ uuid ]
+  description:
+    description:
+    - The description of the syncE Interface Policy.
+    type: str
+  admin_state:
+    description:
+    - The administrative state of the syncE Interface Policy.
+    - The default value is disabled.
+    type: str
+    choices: [ enabled, disabled ]
+    default: disabled
+  sync_state_msg:
+    description:
+    - The sync state message of the syncE Interface Policy.
+    - The default value is disabled.
+    type: str
+    choices: [ enabled, disabled ]
+    default: disabled
+  selection_input:
+    description:
+    - The selection input of the syncE Interface Policy.
+    - The default value is disabled.
+    type: str
+    choices: [ enabled, disabled ]
+    default: disabled
+  src_priority:
+    description:
+    - The source priority of the syncE Interface Policy.
+    - The value must be an integer between 1 and 254.
+    - The default value is 100.
+    type: int
+  wait_to_restore:
+    description:
+    - The wait to restore of the syncE Interface Policy.
+    - The value must be an integer between 0 and 12.
+    - The default value is 5.
+    type: int
```

Comment:
Is it possible to set the custom value (10) when the `selection_input` is disabled?

Reply:
yes on API, but on UI this is not possible. (Trying to keep it as close to UI as possible.